website/src/hugo/content/changelog.md

```diff
@@ -1837,7 +1837,7 @@ This release is identical to v0.19.0-M4.  We mistagged it.  Please proceed to th
   implementors, with more to follow.  Added an `HttpCodecLaws` to http4s-testing.
 * `Uri.Scheme` is now its own type instead of a type alias.
 * `TransferCoding` is no longer a case class. Its `coding` member is now a
-  `String`, not a `CaseInsensitiveString`. Its companion is no longer a
+  `String`, not a `CIString`. Its companion is no longer a
```

Comment:
I guess these historical changes should not refer to the new name,

Reply:
Oh, good catch.  You caught my overzealous search and replace.